docs/platform/feature-flags/stale-feature-notifications.md

```diff
@@ -11,8 +11,17 @@ Stale Features are identified based on specific conditions associated with their
 
 If a Feature is marked as stale, DevCycle encourages users to take action by updating its status to Complete or Archived. This indicates that a Feature is ready to be cleaned up and removed from your codebase. 
 
+
 ## Stale Feature Reasons 
 
+:::note
+Stale Feature detection relies on Variable evaluations triggered through the `.variable()` or `variableValue` SDK call. If your implementation uses `.allVariables()` or `.allFeatures()`, these calls do **not** generate evaluation data. As a result, Features may appear **Unused** and be marked stale even if they are actively used in your code.
```

Comment:
nit: different format between `.variable()` and `variableValue`.. use the same format of: `.variableValue()`?

```suggestion
Stale Feature detection relies on Variable evaluations triggered through the `.variable()` or `.variableValue()` SDK call. If your implementation uses `.allVariables()` or `.allFeatures()`, these calls do **not** generate evaluation data. As a result, Features may appear **Unused** and be marked stale even if they are actively used in your code.
```